config/virtualdesktop/config.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package virtualdesktop
+
+import (
+	"github.com/upbound/upjet/pkg/config"
```

Comment:
Many of the errors stem from this, we recently moved upjet to crossplane. They will be fixed after this change.

```suggestion
	"github.com/crossplane/upjet/pkg/config"
```